client/src/components/Button/Button.stories.js

```diff
@@ -1,7 +1,23 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
-
-import Button from './Button';
+import { storiesOf } from '@storybook/react'
+import Button from './Button'
+import './Button.css'
 
 storiesOf('Button', module)
-    .add('Primary', () => <Button>This is a Button</Button>)
+    .add('light pink button', () => <Button label='Book Now' type='light-pink' /> )
+
+    .add('dark pink button', () => <Button label='DONE' type='dark-pink' /> )
+
+    .add('small grey button', () => <Button label='Next' type='small-grey' /> )
+
+    .add('large red button', () => <Button label='Browse Dresses' type='large-red' large /> )
```

Comment:
This type should maybe be `red` instead of `large-red`. Also, I think `color` is a better name for this prop